fe/fe-core/src/main/java/org/apache/doris/planner/RangePartitionPruner.java

```diff
@@ -75,19 +75,21 @@ public RangePartitionPruner(Map<Long, PartitionItem> rangeMap,
         Column keyColumn = partitionColumns.get(columnIdx);
         PartitionColumnFilter filter = partitionColumnFilters.get(keyColumn.getName());
         if (null == filter) {
-            minKey.pushColumn(LiteralExpr.createInfinity(Type.fromPrimitiveType(keyColumn.getDataType()), false),
-                    keyColumn.getDataType());
-            maxKey.pushColumn(LiteralExpr.createInfinity(Type.fromPrimitiveType(keyColumn.getDataType()), true),
-                    keyColumn.getDataType());
+            int pushMinCount = minKey.fillWithInfinity(partitionColumns, false);
```

Comment:
If there is no condition in the first column, should  we directly add the infinity range of the first column, right?

Reply:
> If there is no condition in the first column, should we directly add the infinity range of the first column, right?

Yes, it might be true. If the columnIdx is 0, we only need to add an infinity range of the first column. there is no need to add the infinity range for all columns. Thanks for your review !

Reply:
> If there is no condition in the first column, should we directly add the infinity range of the first column, right?

Done.